packages/progress-indicator/package-lock.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
We don't save `package.lock` files

Reply:
I will remove this

Reply:
We could add package-lock.json files to the .gitignore file:
`/packages/*/package-lock.json`